03week/PigLatinGui/pigLatin.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+
+function pigLatin() {
+  let newWord = document.getElementById("input").value;
+  
+  newWord = newWord.trim().toLowerCase();
+  let vowels = /[aeiou]/;
+  let finish;
+
+
+  if(newWord[0].match(vowels)){
+    finish = newWord + 'yay';
+  }
+
+  else {
+    let vowelIndice = newWord.indexOf(newWord.match(vowels)[0]);
+    newWord = newWord.trim().toLowerCase().substr(vowelIndice) + newWord.trim().toLowerCase().substr(0, vowelIndice);
```

Comment:
not sure if you needed to .trim and .toLowerCase again since you did that already on line 7